Model/Transaction.php

```diff
@@ -373,6 +373,13 @@ protected function getParentAmounts($attr, $items, $type = 'order') {
      */
     protected function getProductTaxCode($item, $order)
     {
+        // Check for a PTC saved to the Item
+        if ($item->getTjPtc()) {
+            // Return the PTC, or an empty string if the TAXABLE_TAX_CODE is present
+            return $item->getTjPtc() != $this->taxjarConfig::TAXJAR_TAXABLE_TAX_CODE ?: '';
```

Comment:
@ryanreeves-taxjar Does this line return a boolean value instead of the PTC or taxable tax code?

Reply:
No, it's a shorthand ternary operator.